src/main/scala/mesosphere/marathon/core/launchqueue/impl/RateLimiter.scala

```diff
@@ -69,10 +77,16 @@ private[launchqueue] class RateLimiter(clock: Clock) {
 }
 
 private object RateLimiter {
+  /**
+    * The viability threshold may be tuned later, once the mechanism
+    * has been validated.
+    */
+  private val minimumTaskExecutionSeconds: Long = 60
```

Comment:
I'd make this a hidden flag, see for example: https://github.com/mesosphere/marathon/blob/master/src/main/scala/mesosphere/marathon/core/launchqueue/LaunchQueueConfig.scala